source/includes/_webhooks.md

```diff
@@ -0,0 +1,380 @@
+# Webhooks
+
+Our webhooks feature allows a URL, supplied by you, to receive HTTP POST
+requests corresponding to events related to your vote(s). For example, when
+individual votes are recorded.
+
+The URL is configured through the Electionbuddy website, if enabled on your
+account. To enable webhooks contact an Electionbuddy customer support
+representative.
+
+Once enabled, individual webhooks can be enabled for all elections, or enabled or disabled
+on an election-by-election basis, in the organization settings.
+
+![a UI for configuring overall webhook settings](webhook-organization-settings.png)
+
+If enabled on an election-by-election basis, the webhooks can be configured on
+the voter list section fo the vote setup.
+
+![a UI for configuring webhook settings for a single vote](webhook-vote-settings.png)
+
+## Payload Overview
+
+```json
+{
+    "organization_id": 1234,
+    "organization_name": "My organization",
+    "organization_owner_name": "Jane Doe",
+    "organization_owner_email": "support@electionbuddy.com",
+    "election_id": 12345,
+    "election_name": "My election",
+    "identifier": "1",
+    "email": "voter@example.com",
+    "sms": "555-5555",
+    "postal": "1, 3984 Massachusetts Avenue, Washington DC",
+    "status": [
+        "Voted",
+        "Key Surfaced"
+    ],
+    "opened_at": 1600000000,
+    "completed_at": 1600001234,
+    "added_at": null,
+    "spoiled_at": null,
+    "spoil_reason": null,
+    "key_surfaced_at": null,
+    "ballot_printed_at": null,
+    "ip": "0.0.0.0",
+    "unsubscribed_at": null
+    // Non-exhaustive.
+}
+```
+
+The HTTP POST requests will contain a JSON payload. The payload contains
+information about your vote, the organization associated with that vote,
+and the current status of the vote. The parts of the payload will be
+different depending on your vote configuration.
```

Comment:
@Ryan1729 I think it's better to use the term `election` here:

```suggestion
different depending on your election configuration.
```

Reply:
Fair enough. I suppose in this case it is ambiguous whether a single ballot/voter's decision or the entire election is meant here, if we use vote.